packages/oas-to-snippet/__tests__/index.test.js

```diff
@@ -27,28 +29,7 @@ const operation = {
 const formData = { path: { id: 123 } };
 
 test('should be able to accept a har override', () => {
-  const harOverride = {
-    log: {
-      entries: [
-        {
-          request: {
-            method: 'GET',
-            url: 'https://dash.readme.io/api/v1/categories/',
-            httpVersion: 'HTTPS/1.1',
-            headers: [
-              {
-                name: 'authorization',
-                value: 'Basic xxx',
-              },
-            ],
-            queryString: [],
-          },
-        },
-      ],
-    },
-  };
-
-  const codeSnippet = generateCodeSnippet(oas, operation, {}, {}, 'node', oasUrl, harOverride);
+  const codeSnippet = generateCodeSnippet(null, null, null, null, 'node', null, harExamples.full);
```

Comment:
Made all of these arguments `null` because the only ones that should be getting picked up are the language and override.